bundles/AdminBundle/Resources/public/js/pimcore/object/fieldlookup/helper.js

```diff
@@ -0,0 +1,248 @@
+                    if(dataEntry.nodeType == "object") {
+                        baseNode.expand();
+                    } else {
+                        baseNode.collapse();
+                    }
+                }
+            }
+        }
+    },
+
+
+    addDataChild: function (type, initData, attributePrefix, showFieldname, ctx, dataContext) {
+
+        if(type != "objectbricks" && !initData.invisible) {
+            var isLeaf = true;
+
+            if(type == "localizedfields") {
+                isLeaf = false;
+            }
+
+            var key = initData.name;
+            if(attributePrefix) {
+                key = attributePrefix + "~" + key;
+            }
+
+            var text = t(initData.title) + " (" + key.replace("~", ".") + ")";
+
+            var newNode = {
+                originalText: text,
+                text: text,
+                key: key,
+                type: "data",
+                layout: initData,
+                leaf: isLeaf,
+                dataType: type,
+                iconCls: "pimcore_icon_" + type,
+                qtip: ctx.object.data.data[key],
+                dataContext: dataContext,
+                fieldtype: initData.fieldtype
+            };
+
+            newNode = this.appendChild(newNode);
+
+            if (newNode.parentNode.data && newNode.parentNode.data.layout
+                && newNode.parentNode.data.layout.fieldtype == "localizedfields"
+                && newNode.parentNode.data.layout.fieldtype == "fieldcollections"
+                && newNode.parentNode.data.layout.fieldtype == "objectbricks"
+
+                ) {
+                newNode.disabled = true;
+            }
+
+            this.expand();
+            return newNode;
+        } else {
+            return null;
+        }
+
+    },
+
+    updateFilter: function (tree, filterField) {
+
+        tree.getStore().clearFilter();
+        var currentFilterValue = filterField.getValue().toLowerCase();
+
+        tree.getStore().filterBy(function (item) {
+            if (item.data.type === "data") {
+                var text = t(item.data.originalText);
+                let fieldtype = item.data.fieldtype;
+
+                if (fieldtype == "localizedfields"
+                    || fieldtype == "fieldcollections"
+                    || fieldtype == "objectbricks") {
+                    return true;
+                }
+
+                if (currentFilterValue) {
+                    var textLower = text.toLowerCase();
+                    var idx = textLower.indexOf(currentFilterValue);
+                    if (idx == -1) {
+                        return null;
+                    }
+                    var pre = text.substring(0, idx);
+                    var post = text.substring(idx + currentFilterValue.length);
+                    var match = text.substring(idx, idx + currentFilterValue.length);
+                    text = pre + "<font color='#00AA00'><b color='green'>" + match + "</b></font>" + post;
```

Comment:
`<font color='#00AA00'><b color='green'>` What? :D

Reply:
This is 1:1 from the existing bundle (it. highlights the filter match). Suggestions for better styling are welcome, either as part of this PR or afterwards :-)

Reply:
What about `<strong style="color: #00AA00">` considering the state of [\<font\>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font).

Reply:
changed it but I am wondering if we should replace this greenish highlight color with something else that causes less pain :-)

Reply:
What about  \#008040 (https://jsfiddle.net/10yogmak/1/)? Should hit most requirements for [color contrast](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrasts).

I first tried \#28a050 (the background colour on the Save & Publish button), but that one had a little too low contrast versus light colours at (3.37, 2.48, 5.61).

Reply:
thx! looks definitely better

![image](https://user-images.githubusercontent.com/6142086/78993509-1e9cc200-7b3e-11ea-9305-bfea2a8193b3.png)